test/unit/atomic/src/AtomicTest.cpp

```diff
@@ -354,6 +354,32 @@ class AtomicTestKernel
     }
 };
 
+#if defined(ALPAKA_ACC_GPU_CUDA_ENABLED) || defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+//#############################################################################
+// Skip all atomic tests for the unified CUDA/HIP backend.
+// CUDA and HIP atomics will be tested separate.
+template<
+    typename T,
+    typename TDim,
+    typename TIdx>
+class AtomicTestKernel<
+    alpaka::acc::AccGpuUniformCudaHipRt<TDim, TIdx>,
+    T>
+{
+public:
+    //-----------------------------------------------------------------------------
+    ALPAKA_NO_HOST_ACC_WARNING
+    ALPAKA_FN_ACC auto operator()(
+        alpaka::acc::AccGpuUniformCudaHipRt<TDim, TIdx> const & acc,
+        bool * success,
+        T operandOrig) const
```

Comment:
The compiler complains about unused parameters parameters. Please use `alpaka::ignore_unused`.

Reply:
fixed